aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -21,11 +21,11 @@ For apps running on .NET Core 2.2 or later, the module supports an in-process ho
 
 In-procsess hosting is opt-in for existing apps, but [dotnet new](/dotnet/core/tools/dotnet-new) templates default to the in-process hosting model for all IIS and IIS Express scenarios.
 
-To configure an app for in-process hosting, add the `<AspNetCoreModuleHostingModel>` property to the app's project file with a value of `inprocess` (out-of-process hosting is set with `outofprocess`):
+To configure an app for in-process hosting, add the `<AspNetCoreHostingModel>` property to the app's project file (for example, MyApp.csproj) with a value of `inprocess` (out-of-process hosting is set with `outofprocess`):
```

Comment:
The file should be emphasized (asterisks) ... \*MyApp.csproj\*. Note tho that we've assumed in most of our docs that when we say "project file" that the reader knows what we mean. Therefore, we don't usually provide a named file example. It's cool tho if this makes sense here as Sourabh said it does. :+1: